<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="4" total="10" passed="2" failed="1" skipped="3">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-02T08:17:27 IST" name="Suite" finished-at="2024-09-02T08:17:50 IST" duration-ms="23118">
    <groups>
      <group name="login">
        <method signature="Login.chkuserid()[pri:1, instance:testcases.Login@201a4587]" name="chkuserid" class="testcases.Login"/>
        <method signature="Login.chkpassword()[pri:2, instance:testcases.Login@201a4587]" name="chkpassword" class="testcases.Login"/>
        <method signature="Login.chkuserid()[pri:1, instance:testcases.Login@2892d68]" name="chkuserid" class="testcases.Login"/>
        <method signature="Login.chkpassword()[pri:2, instance:testcases.Login@2892d68]" name="chkpassword" class="testcases.Login"/>
      </group> <!-- login -->
    </groups>
    <test started-at="2024-09-02T08:17:27 IST" name="AdminTest" finished-at="2024-09-02T08:17:27 IST" duration-ms="64">
      <class name="testcases.Login">
        <test-method is-config="true" signature="beforeTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Login@201a4587]" started-at="2024-09-02T08:17:27 IST" name="beforeTest" finished-at="2024-09-02T08:17:27 IST" duration-ms="10" status="SKIP">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://www.amazon.in]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[LoginReport]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[BrowserTest]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[false]]>
              </value>
            </param>
          </params>
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skip test BrowserTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skip test BrowserTest
at core.Page.beforeTest(Page.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="chkuserid()[pri:1, instance:testcases.Login@201a4587]" started-at="2024-09-02T08:17:27 IST" name="chkuserid" finished-at="2024-09-02T08:17:27 IST" duration-ms="0" status="SKIP">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skip test BrowserTest]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skip test BrowserTest
at core.Page.beforeTest(Page.java:38)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)
at org.testng.TestRunner.beforeRun(TestRunner.java:627)
at org.testng.TestRunner.run(TestRunner.java:589)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- chkuserid -->
        <test-method signature="chkpassword()[pri:2, instance:testcases.Login@201a4587]" started-at="2024-09-02T08:17:27 IST" name="chkpassword" finished-at="2024-09-02T08:17:27 IST" depends-on-methods="testcases.Login.chkuserid" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Login.chkpassword()[pri:2, instance:testcases.Login@201a4587] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Login.chkpassword()[pri:2, instance:testcases.Login@201a4587] depends on not successfully finished methods
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- chkpassword -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:testcases.Login@201a4587]" started-at="2024-09-02T08:17:27 IST" name="afterTest" finished-at="1970-01-01T05:30:00 IST" duration-ms="-1725245247540" status="SKIP">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- testcases.Login -->
      <class name="testcases.Shop">
        <test-method signature="searchProducts(java.lang.String)[pri:1, instance:testcases.Shop@1151e434]" started-at="2024-09-02T08:17:27 IST" name="searchProducts" depends-on-groups="login" finished-at="2024-09-02T08:17:27 IST" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Shop.searchProducts(java.lang.String)[pri:1, instance:testcases.Shop@1151e434] depends on not successfully finished methods in group "login"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Shop.searchProducts(java.lang.String)[pri:1, instance:testcases.Shop@1151e434] depends on not successfully finished methods in group "login"
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:102)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchProducts -->
      </class> <!-- testcases.Shop -->
    </test> <!-- AdminTest -->
    <test started-at="2024-09-02T08:17:27 IST" name="ForgetTest" finished-at="2024-09-02T08:17:50 IST" duration-ms="22969">
      <class name="testcases.Login">
        <test-method is-config="true" signature="beforeTest(java.lang.String,java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testcases.Login@2892d68]" started-at="2024-09-02T08:17:27 IST" name="beforeTest" finished-at="2024-09-02T08:17:31 IST" duration-ms="3893" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[edge]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[https://www.amazon.in]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[ForgetReport]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[BrowserTest]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[true]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="chkuserid()[pri:1, instance:testcases.Login@2892d68]" started-at="2024-09-02T08:17:31 IST" name="chkuserid" finished-at="2024-09-02T08:17:32 IST" duration-ms="1103" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- chkuserid -->
        <test-method signature="chkpassword()[pri:2, instance:testcases.Login@2892d68]" started-at="2024-09-02T08:17:32 IST" name="chkpassword" finished-at="2024-09-02T08:17:47 IST" depends-on-methods="testcases.Login.chkuserid" duration-ms="15126" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[error coming when password is valid..]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: error coming when password is valid..
at org.testng.Assert.fail(Assert.java:99)
at testcases.Login.chkpassword(Login.java:58)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- chkpassword -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:testcases.Login@2892d68]" started-at="2024-09-02T08:17:48 IST" name="afterTest" finished-at="2024-09-02T08:17:50 IST" duration-ms="1673" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- testcases.Login -->
      <class name="testcases.ForgetTest">
        <test-method signature="forget()[pri:0, instance:testcases.ForgetTest@5ab956d7]" started-at="2024-09-02T08:17:47 IST" name="forget" finished-at="2024-09-02T08:17:48 IST" duration-ms="1161" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- forget -->
      </class> <!-- testcases.ForgetTest -->
    </test> <!-- ForgetTest -->
  </suite> <!-- Suite -->
</testng-results>
